src/applications/vaos/new-appointment/components/DateTimeSelectPage/WaitTimeAlert.jsx

```diff
@@ -20,7 +21,14 @@ export const WaitTimeAlert = ({
 }) => {
   const today = moment();
   const momentPreferredDate = moment(preferredDate);
-  const momentNextAvailableDate = moment(nextAvailableApptDate);
+  let momentNextAvailableDate;
+  if (nextAvailableApptDate?.includes('Z')) {
```

Comment:
I'm probably going to add a ticket to update our code to always return slots in UTC. We have several instances of this logic now, because slots are different between v0 and v2.